source/utilities.tex

```diff
@@ -13950,10 +13950,11 @@
 \begin{itemdescr}
 \pnum
 \returns
-A new \tcode{scoped_allocator_adaptor} object where each allocator \tcode{A} in the
-adaptor is initialized from the result of calling
-\tcode{allocator_traits<A>::select_on_container_copy_construction()} on the
-corresponding allocator in \tcode{*this}.
+A new \tcode{scoped_allocator_adaptor} object
+where each allocator \tcode{a} of type \tcode{A} within the adaptor
+is initialized with
+\tcode{allocator_traits<A>::select_on_container_copy_construction(a2)},
+where \tcode{a2} is the corresponding allocator in \tcode{*this}.
```

Comment:
I'm finding it hard not to read "each allocator `a` of type `A`" as meaning this only applies to the ones of type `A`, for some specific type `A`.

How about:


```suggestion
A new \tcode{scoped_allocator_adaptor} object
where each allocator \tcode{a} within the adaptor
is initialized with
\tcode{allocator_traits<A>::select_on_container_copy_construction(a2)},
where \tcode{A} is the type of \tcode{a} and \tcode{a2} is the corresponding allocator in \tcode{*this}.
```

Reply:
You're right. Fixed.

Reply:
How about instead of `a` and `a2` we have `a1` and `a2` symmetrically?

Otherwise good, thanks!

Reply:
That seems fine too.

I considered something fancy like `A`<sub>n</sub>, `a`<sub>n</sub> and `this->a`<sub>n</sub>, but it wouldn't be right as `this->a`<sub>n</sub> isn't actually a member of `*this`.

Reply:
@jensmaurer: I'd somewhat prefer `a1`, `a2` over `a`, `a2`, but if you feel very strongly, we can do it your way. Please feel free to merge either version!

Reply:
@jensmaurer: For your consideration, but fine to go in in principle.

Reply:
@tkoeppe, change to a1, a2. Please double-check.

Reply:
Thanks, I like it! Can we also make the type of `a1` be `A1`?

Reply:
@tkoeppe, done.

Reply:
Thank you!